### DISCUSSION TITLE
之前builld过主分支的TVM，请问这样的话pip mlc-ai-nightly也无法import relay

### DISCUSSION THREAD

**Discussion by eAzure** [mlc-ai/mlc-zh#107] [Q&A] [answered]
之前简单使用过TVM，用的是主分支上的，相关的环境变量也进行了设置，但是在本课程中使用python -m pip install mlc-ai-nightly -f https://mlc.ai/wheels 也无法from tvm import relay，请问是不是需要重新对TVM进行build

**Comment by Hzfengsy** [comment, marked as answer]
可以考虑临时删除环境变量 `unset $PYTHONPATH`

**Reply by eAzure** [reply]
明白了，谢谢
